AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/Request/AuthSignUpRequest+Validation.swift

```diff
@@ -17,14 +17,5 @@ extension AuthSignUpRequest {
                 AuthPluginErrorConstants.signUpUsernameError.recoverySuggestion, nil)
             throw error
         }
-
-        guard let password = password,
```

Comment:
By removing the whole block, we will no longer fail when a `nil` password is provided.
Is this expected/accepted?

Reply:
Yeah... The service will throw an error. Less logic on the client, less bugs and unforeseen behaviour.